pkg/server/container_create_windows.go

```diff
@@ -497,6 +498,19 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 	return g.Config, nil
 }
 
+func (c *criService) addOCIWindowsDevices(g *generator, devs []*runtime.Device) {
```

Comment:
It looks like this can just be a free function instead of receiving a `*criService`.

Reply:
Sure can. Is there a reason to prefer that over this? I have no issue making the change, just curious.

Reply:
Generally if it's not logically tied to the receiver, and doesn't use the receiver at all, I think it should be a free function instead. It makes it easier to reason about the code since it changes what values the function has access to.